www/templates/diary/list.html

```diff
@@ -86,12 +86,16 @@
                     </div>
                 </div>
              </div>
-             <div class="row" style="padding-left: 5px;padding-right: 5px;">
+             <div 
+                ng-if="isAnalyzed(tripgj)"
```

Comment:
let's avoid calling functions from the HTML. It significantly reduces performance because of the way in which angular . Create a `$scope.isAnalyzed` variable and use it instead

See https://github.com/e-mission/e-mission-phone/commit/30561b94d8691e37fc3e50b8488f5da82f6dd815 for the change which removed function based conditionals.

Reply:
Patched in https://github.com/e-mission/e-mission-phone/pull/798/commits/b57742a68746273b62a4b3bbe91b76d209d365c4.